app/components/password_confirmation_component.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= content_tag(:'lg-password-confirmation', **tag_options) do %>
+  <%= render ValidatedFieldComponent.new(
+        form: form,
+        name: :password,
+        type: :password,
+        label: default_label,
+        required: true,
+        **field_options,
+        input_html: field_options[:input_html].to_h.merge(
+          id: input_id,
+          class: ['password-confirmation__input1', *field_options.dig(:input_html, :class)],
+        ),
+      ) %>
+  <%= render ValidatedFieldComponent.new(
+        form: form,
+        name: :password_confirmation,
+        type: :password_confirmation,
```

Comment:
I think this will need to be feature flagged in some manner since it will be backwards incompatible when running alongside a version that does not require password_confirmation.